bin/development-server.js

```diff
@@ -67,13 +65,11 @@ app.get("/get", function(req, res) {
     body => res.json(JSON.parse(body))
   );
 
-  httpReq.on('error', function (err) {
-    res.status(500).send(err.code);
-  });
-})
+  httpReq.on("error", err => res.status(500).send(err.code));
+});
 
 // Listen
-app.listen(8000, "localhost", function(err, result) {
+app.listen(8000, "0.0.0.0", function(err, result) {
```

Comment:
That's fine with me. Does anyone use this to send a public URL to someone else? Usually you have to use a forwarding service anyway because you are behind a firewall/router.